ParabolInc/parabol
# feat(ad-hoc): Add create team dialog

Fixes #8827
<img width="712" alt="image" src="https://github.com/ParabolInc/parabol/assets/466991/de5450ef-b295-4c5a-8f0a-c02c489c76a8">

**Not included:** 
- Email invites expires warning

**How to test:**

- add `adHocTeam` user-level feature flag
- go to any meeting in activity library, open teams dropdown
- create a team from teams dropdown by clicking "add team"
- see that after creating a team it is automatically selected, invites are sent and you can successfully 
- test with different set of users and custom emails
- See if you selected users from different orgs an org selection dropdown appears showing all potential orgs correctly
- See if you entered only custom emails, org selection dropdown appears and you can select any of your org
